### DISCUSSION TITLE
Clone Repository

### DISCUSSION THREAD

**Discussion by Jayantadeb73** [community/community#23812] [New to GitHub]
<p>Hi All,</p>
<p>I have successfully cloned one repository using ‘git clone &lt;repo_url&gt;’.</p>
<p>When i was tring to clone the same repo again, geting below error message :</p>
<p>fatal : destination path &lt;repo_name&gt; already exists and is not an empty directory.</p>
<p>Now my question is, do i need to remove the local repository (directory) before cloning that same repo once again?</p>
<p>Thanks with Regards,</p>
<p>JD</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/jayantadeb73">@jayantadeb73</a></p>
<p>You could use:<br>
git clone &lt;url-link&gt; &lt;name-repo&gt;</p>
<p>With this you can rename the cloned repository with new name.</p>
<p>Hope this help</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>

**Comment by ajay3699** [comment]
<p><span class="hashtag">#git</span> clone <a href="https://github.com/pure-LOGIC/Instagram.git" rel="nofollow noopener">https://github.com/pure-LOGIC/Instagram.git</a> fatal: destination path ‘Instagram’ already exists and is not an empty directory. this is my problem but it is not solving by using ur commmand please help</p>

**Comment by Wabri** [comment]
<p>Giving a name to the clone may solve your problem, try this one:</p>
<pre><code>git clone[https://github.com/pure-LOGIC/Instagram.git](https://github.com/pure-LOGIC/Instagram.git)InstagramPureLogic
</code></pre>
<p>-Gabriele-</p>

**Comment by airtower-luna** [comment]
<p>Do you actually want to clone again, or just update the clone you already have? In the latter case you should use “git pull” instead.</p>

**Comment by ssharma006** [comment]
![Screenshot 2025-07-07 182006](https://github.com/user-attachments/assets/1979b53b-076b-4fb4-975a-14d097b4f08b)
I am getting this error i am fresher not much aware of Python or github

**Reply by airtower-luna** [reply]
This has nothing to do with cloning a repository, you're just missing a space. Add it between the mode and the file path.

**Reply by ssharma006** [reply]
can you send me the code in right form or link from where I can copy it .  thnks
